README.md

```diff
@@ -15,9 +15,10 @@ slurmutils package include:
 
 #### `from slurmutils.editors import ...`
 
+* `acctgatherconfig`: An editor for _acct_gather.conf_ configuration files.
```

Comment:
Nit/thought: if the file edited is called `acct_gather.conf`, shouldn't we use `acct_gatherconfig` as the name of the method for consistency? It does look a bit weird to have `config` attached to only `gather` though...

Maybe we should add an underscore between the editor name and the "config" on all editors?

Reply:
My vote would be for `acctgatherconfig`. My preference is avoiding underscores unless needed for legibility, generally.

Reply:
We actually discussed this on #35 :smile: 

We decided it would be better to have it be `acctgatherconfig` instead of `acct_gatherconfig` as it read awkwardly. It's encouraged in [PEP008](https://peps.python.org/pep-0008/#package-and-module-names) to have module names be all lowercase, and underscores should only be used to improve readability. We're still 0.x, so this could change in the future, but at least for now, I like not having to press the shift key when importing a module :sweat_smile: